python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/system.j2

```diff
@@ -25,12 +25,16 @@ system control-plane
 {%         set with_vrf_default = system.control_plane.ipv4_access_groups | selectattr('vrf', 'arista.avd.defined') | selectattr('vrf', 'equalto', 'default') | arista.avd.natural_sort %}
 {%         set sorted_ipv4_access_groups =  without_vrf | list + with_vrf_default | list + with_vrf_non_default | list %}
 {%     endif %}
-{%     for acl_set in sorted_ipv4_access_groups | arista.avd.default([]) %}
-{%         set cp_ipv4_access_grp = "ip access-group " ~ acl_set.acl_name %}
-{%         if acl_set.vrf is arista.avd.defined %}
-{%             set cp_ipv4_access_grp = cp_ipv4_access_grp ~ " vrf " ~ acl_set.vrf %}
+{%     for acl_set in system.control_plane.ipv4_access_groups | arista.avd.natural_sort %}
```

Comment:
can you fix the j2 template for vrf default and none case. So if vrf is default or none then we have to render the only last command.
Like from your example
ipv4_access_groups:
      - acl_name: "acl4_1"
      - acl_name: "acl4_3"
        vrf: default
it should render the only below config:
ip access-group acl4_3 in

Reply:
We can as well optionally simply pick first IPv4 and IPv6 out of all with either "missing" VRF or those where it is set to `default`. EoS takes last line from overlapping commands pushed to `conf` or `conf session` because it simply overwrites previous ones. As we generate designed configuration state for the device - we should probably avoid generating commands which would be overwriting each other (although EoS would be OK with it).